databases/week1/databases_week1_homework.sql

```diff
@@ -0,0 +1,37 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT( * )
+FROM task;
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT *
+FROM task
+WHERE due_date IS NULL;
+-- Find all the tasks that are marked as done
+SELECT task.*, status.name
+FROM task INNER JOIN status ON task.user_id = status.id
```

Comment:
Hey @Mekdes1, please make sure the JOIN is correct. Same for the next query

```suggestion
FROM task INNER JOIN status ON task.status_id = status.id
```